src/recon_buildblock/BackProjectorByBin.cxx

```diff
@@ -200,11 +200,12 @@ BackProjectorByBin::back_project(const ProjData& proj_data, int subset_num, int
                                          subset_num, num_subsets);
 
 #ifdef STIR_OPENMP
-#pragma omp parallel shared(proj_data, symmetries_sptr)
-#endif
-  {
-#ifdef STIR_OPENMP
-#pragma omp for schedule(dynamic)
+  #if _OPENMP <201107
+    #pragma omp parallel for  shared(proj_data, symmetries_sptr) schedule(dynamic)
+  #else
+    // OpenMP loop over both vs_nums_to_process and tof_pos_num
+    #pragma omp parallel for  shared(proj_data, symmetries_sptr) schedule(dynamic) collapse(2)
+  #endif
```

Comment:
I also changed a bit of the OpenMP `#ifdef STIR_OPENMP` in `BackProjectorByBin` here so that it matches `ForwardProjectorByBin`